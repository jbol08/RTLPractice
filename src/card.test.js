import { render } from "@testing-library/react";
import Card from './Card';

//smoke test
it('renders the div without crashing', () => {
    render (<Card />)
})

//snapshot test
it('takes a snapshot of the initial setup', () => {
    const { asFragment } = render(<Card />);
    expect(asFragment()).toMatchSnapshot()
})
