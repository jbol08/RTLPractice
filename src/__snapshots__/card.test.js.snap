// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`takes a snapshot of the initial setup 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    />
    <img
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;
